torch/_dynamo/variables/tensor.py

```diff
@@ -227,6 +231,10 @@ def var_getattr(self, tx, name):
             result = ConstantVariable(self.is_quantized, **options)
         elif name == "is_sparse" and self.is_sparse is not None:
             result = ConstantVariable(self.is_sparse, **options)
+        elif name == "storage_offset":
+            return variables.LambdaVariable(
+                lambda *args, **kwargs: ConstantVariable(self.storage_offset)
```

Comment:
is **options missing here relative to the call_method impl returning `ConstantVariable(self.storage_offset, **options)` below?

Reply:
LambdaVariable covers this!

LambdaVariable here gets a  `.add_options(self)`, And its call_function passes options down:

```
    def call_function(
        self, tx, args: "List[VariableTracker]", kwargs: "Dict[str, VariableTracker]"
    ) -> "VariableTracker":
        return self.fn(*args, **kwargs).add_options(self)
```